src/pages/resultsView/mutation/Mutations.tsx

```diff
@@ -65,6 +73,14 @@ export default class Mutations extends React.Component<
             .props.store.mutationsTabFilteringSettings.excludeGermline;
     }
 
+    @action
+    public setSelectedGeneSymbol(hugoGeneSymbol: string) {
+        this.selectedGeneSymbol = hugoGeneSymbol;
```

Comment:
these two lines duplicate the source of truth, we should not need to keep the hugoGeneSymbol in both url and as instance property of the Mutations component.